marlowe-runtime.nix

```diff
@@ -31,35 +33,52 @@ let
     };
   };
 in {
+  # Please upgrade per netowk instances in "one go".
+  # Let's say that we have a single `mainnet` runtime if we:
+  # - remove that instance
+  # - redeploy
+  # - add new mainnet instance
+  # - redeploy
+  # Then the cardano-node behind this network will be destroyed and recreated and resynced.
```

Comment:
State won’t be destroyed, the node will just not be running.

Reply:
Okey. Cool! In order to really remove the state then we should proceed manually, yes?

Reply:
Yep